include/geos/operation/polygonize/PolygonizeGraph.h

```diff
@@ -130,11 +130,10 @@ class GEOS_DLL PolygonizeGraph: public planargraph::PlanarGraph {
 	 */
 	void deleteDangles(std::vector<const geom::LineString*> &dangleLines);
 
-private:
-
-	static int getDegreeNonDeleted(planargraph::Node *node);
+ private:
+	int getDegreeNonDeleted(planargraph::Node *node) const;
 
-	static int getDegree(planargraph::Node *node, long label);
+	int getDegree(planargraph::Node *node, long label) const;
```

Comment:
Were these intentional ? Changed from static to non-static methods

Reply:
Intentional